HomeWork_09/HW9.py

```diff
@@ -0,0 +1,15 @@
+text = int(input("Enter a digit: "))
+
+def my_generator(text):
```

Comment:
Your generator should yield fibonacci numbers but not a range from 0 to input value. Please, update your solution